app/build.gradle

```diff
@@ -83,6 +83,10 @@ android {
         // Didn't find a way to fix this, but it does not seem to cause any issue as we've been
         // using the learning for a while
         disable 'ClickableViewAccessibility'
+
+        // Most unused resources have been removed, but the removing the representative pictures
+        // seems to break the app, so these have been kept
+        disable 'UnusedResources'
```

Comment:
That's not necessary: it's not that it "seems to break the app" in some unexplained way, it's very much explained by [this](https://github.com/HumanLearning2021/HumanLearningApp/commit/0df1ff843c7f5d12ec39509b4bc886f15e5ecbbe#diff-71c4f73cc25e4063965a0465ecdc48d7307c911deed45f7a46192326bdbc9106R181). (To be fair my comment wasn't very clear)

Please remove this and rebase on main

Reply:
i see thanks, done.